spec/controllers/review_controller_spec.rb

```diff
@@ -88,6 +88,44 @@
 
       required_permissions = ["signin", "GDS Editor", "Unreleased feature", "2i reviewer"]
 
+      describe "GET approve 2i review" do
+        it "can be accessed by the reviewer, if they have GDS editor, Unreleased feature and 2i reviewer permissions" do
+          stub_user.permissions = required_permissions
+          get :approve_2i_review, params: { step_by_step_page_id: step_by_step_page.id }
+
+          expect(response.status).to eq(200)
+        end
+
+        (required_permissions - %w(signin)).each do |required_permission|
```

Comment:
Generally (I did try find a good source but Google failed me, so you'll have to just trust me :) ) it's not considered a best practice to dynamically create expectations at runtime with loops. This would probably be more commonly handled via a shared expectation.

Though I do wonder here what we're actually testing here, I wouldn't have thought we'd need to test every permission returns a 403 as that would be us using the test to confirm that the configuration of permissions in our controller matches the one in the test spec. Whereas I imagine we just need one test that confirms that we can return a 403?